CHANGELOG.md

```diff
@@ -10,7 +10,9 @@
 
 ##### Bug Fixes
 
-* None.
+* Avoid pattern matching false positives in `unused_optional_binding` rule.
```

Comment:
requires two trailing spaces as described in CONTRIBUTING.md: https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes